dist/profile/manifests/l10n_server.pp

```diff
@@ -0,0 +1,45 @@
+#
+# Accept submissions from the translation plugin
+# containerized in https://github.com/jenkins-infra/l10n-server
+class profile::l10n_server (
+  # Parameters supplied by Hiera
+  $image_tag = 'latest',
+) {
+  include profile::docker
+
+  validate_string($image_tag)
+  $user = 'l10n'
+  $image = 'jenkinsciinfra/l10n-server'
+
+  docker::image { $image:
+    image_tag => $image_tag,
+  }
+
+  docker::run { 'l10n':
+    volumes  => ['/home/l10n/data:/var/l10n'
```

Comment:
Since the homedir is duplicated a few places, I suggest making it a variable like `$user` above